public_html/planeObject.js

```diff
@@ -28,7 +28,14 @@ function PlaneObject(icao) {
         this.nav_altitude   = null;
         this.nav_heading    = null;
         this.nav_modes      = null;
-        this.nav_qnh        = null;
+		this.nav_qnh        = null;
```

Comment:
I know the tab/space stuff is all over the place in this code, but can we keep it to spaces here to be consistent with the surrounding stuff?